protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/signal/SignalSubscriptionRecord.java

```diff
@@ -108,6 +108,11 @@ public DirectBuffer getBpmnProcessIdBuffer() {
     return bpmnProcessIdProp.getValue();
   }
 
+  public long getSubscriptionKey() {
```

Comment:
❌ You should add `@JsonIgnore` annotation on top of this method, because we don't want to allow serialization/deserialization for this field